### PR TITLE
Validate that we support launching urls with escaping that is not needed

### PR DESCRIPTION
**Pull request by digeff**
New test that validates that we can hit breakpoints if the url has an escaped character like file:///c%3A/myproject/index.html

Validates the bug fixed by: https://github.com/microsoft/vscode-chrome-debug-core/pull/474

### CODE REVIEW

File: testdata/featuresTests/unusualLaunchJson/urlUsesEscapedCharacter/app.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+function runCode() {
```
Comment (EricCornelson) [inline testdata/featuresTests/unusualLaunchJson/urlUsesEscapedCharacter/app.ts]: Is it necessary to create a new test project just for this test? Could we re-use one of the existing ones?

Comment (digeff) [reply testdata/featuresTests/unusualLaunchJson/urlUsesEscapedCharacter/app.ts]: What disadvantage do you see in this test having it's own test project?

Comment (EricCornelson) [reply testdata/featuresTests/unusualLaunchJson/urlUsesEscapedCharacter/app.ts]: Just excess bloat of the testdata/ dir, and the repo in general. This test really only cares about the launch.json params, and whether or not we can hit that url, so the test project itself is superfluous. We already have a bunch of little test projects that would meet the same need.

Why add more stuff to maintain when we don't need to?

Comment (digeff) [reply testdata/featuresTests/unusualLaunchJson/urlUsesEscapedCharacter/app.ts]: I'm okay with trying to reuse projects. The existing ones have very specific behaviors that we want for the individual tests, so I don't see a great candidate to reuse it. I can move this project to a reusableProjects\ folder so people know this project doesn't have any particular behaviors, and they can reuse it for future tests. What do you think?

Comment (digeff) [reply testdata/featuresTests/unusualLaunchJson/urlUsesEscapedCharacter/app.ts]: Changed to use simple project as discussed offline

**Pull request merged**